backend/src/routes/pathway.ts

```diff
@@ -0,0 +1,32 @@
+import { Router } from 'express';
+import Pathway from 'models/pathway';
+
+const router = Router();
+
+router.get('/', (req, res) => {
+  return Pathway.find({ id: '2TJDsrHux' });
+});
+
+router.put('/:id', (req, res) => {
+  if (req.params.id !== req.body.id) res.status(409).send('Pathway id does not match URL id');
+  Pathway.findOneAndUpdate(
+    { id: req.params.id },
+    req.body,
+    { overwrite: true, new: true },
+    (err, product) => {
+      if (err) res.send(err);
```

Comment:
What status code should the errors returns - 500?

Reply:
I think 500 makes sense for catching all the ones that aren't handled more specifically (like the 404's or 409's).